tests/integration/__init__.py

```diff
@@ -389,6 +389,17 @@ def _id_and_mark(f):
 requires_custom_payload = pytest.mark.skipif(SCYLLA_VERSION is not None or PROTOCOL_VERSION < 4,
                                             reason='Scylla does not support custom payloads. Cassandra requires native protocol v4.0+')
 xfail_scylla = lambda reason, *args, **kwargs: pytest.mark.xfail(SCYLLA_VERSION is not None, reason=reason, *args, **kwargs)
+
+
+def xfail_scylla_version(reason, oss_scylla_version, ent_scylla_version, *args, **kwargs):
+    current_version = Version(get_scylla_version(SCYLLA_VERSION) if SCYLLA_VERSION is not None else '0.0.0')
+    if current_version > Version("2018.1"):
+        lt_scylla_version = Version(ent_scylla_version)
+    else:
+        lt_scylla_version = Version(oss_scylla_version)
+    return pytest.mark.xfail(current_version < lt_scylla_version,
```

Comment:
As I mentioned in old PR, this should be named `xfail_scylla_version_lt` or similar. Current name does not convey the meaning.

Reply:
Also, doesn't this break testing with Cassandra?
If `SCYLLA_VERSION` is not specified, `current_version` will be `0.0.0`.
`lt_scylla_version` will be `oss_scylla_version`, and the comparision will be `0.0.0 < oss_scylla_version`, which will be true and skip the test for Casssandra.

This function should probably return identity decorator if `SCYLLA_VERSION` is not set, or handle it in some other way that does not skip the test for Cassandra.